tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -38,9 +38,35 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/ptr_util.h"
 
+#include "tensorflow/core/lib/io/zlib_compression_options.h"
+#include "tensorflow/core/lib/io/zlib_outputbuffer.h"
+#include "tensorflow/core/platform/env.h"
+
 namespace tensorflow {
 namespace data {
 
+enum CompressionType : int { ZLIB = 0, GZIP = 1, RAW = 2, UNCOMPRESSED = 3 };
+
+// Gets the compression name.
+string CompressionName(CompressionType compression_type);
```

Comment:
Rename the method to `CompressionTypeToString` (or simply `ToString`) and update the documentation to "Returns a string representation for the given compression type."`

Reply:
Done